scripts/productos.js

```diff
@@ -0,0 +1,11 @@
+// JSON
+var objeto = {
+    nombre: "Sunset Crush",
+    autor: ["Cristina Cascajares", "Denisse Signoret", "Fer Lizano"]
+    descripcion: "Beauty Salon & Shop",
+    anio: 2020,
+    lugar: "DevF Remoto",
+}
+obtenerInfo: function() {
+    this.nombre + " fue creada en DevF por " + this.autor + " en el año " + this.anio;
```

Comment:
Recuerda que this.autor es un arreglo y debemos acceder a su indice

`this.nombre + " fue creada en DEV.F por " + this.autor[0] + " "  + this.autor[1] + " "  + this.autor[2] + " en el año " + this.anio;`

Reply:
listo ya lo corregí!